Assets/Scripts/EOSEACLobbyManager.cs

```diff
@@ -233,7 +233,7 @@ private void OnLobbyUpdated()
             }
         }
 
-        private void OnLobbyChanged()
+        private void OnLobbyChanged(LobbyChangedEvent lobbyChangedEvent)
```

Comment:
It could be discussed whether to pass an argument here or to have separate events for each type.
The OnLobbyChanged by its name handles lobby changes regardless how the lobby was changed.
It feels indirect to have, for instance, OnKicked to call OnCurrentLobbyChanged then back to a kicked state handler.
Let us know your thoughts.